### ISSUE TITLE
wintergrasp

### ISSUE THREAD

**Issue by kamila231993** [noisiver/mod-progression#5]
[<a href="https://freeimage.host/i/JWcQtVI"><img src="https://iili.io/JWcQtVI.md.png" alt="JWcQtVI.md.png" border="0"></a><br /><a target='_blank' href='https://freeimage.host/'>upload photos and share link</a><br />](url)

in wintergrasp are these items ok at this vendor in patch 3.0?

**Comment by noisiver** [comment]
No, those are wrong. I forgot to look at vendors found within Wintergrasp before but will look into it now.

**Comment by noisiver** [comment]
Actually, that vendor (and the alliance one) were added in 3.3 so I'm removing them from prior patches right now.

**Comment by noisiver** [comment]
I'm closing this because I just commited a change to hide them until patch 3.3. Should you find anything else strange in Wintergrasp, or any other area, feel free to create a report and I'll look into it.

Thank you for telling me about this one, it's easy to miss things when you're working on it alone and don't have anyone testing it for you but now I have you so thanks.

**Comment by noisiver** [comment]
I realized these are not spawned in the database but I assume spawned by the C++ scripts. I'll be fixing this and will keep this issue open until I'm done.

**Comment by kamila231993** [comment]
aaaa ok

**Comment by noisiver** [comment]
I just commited a fix for this. I hope I didn't miss anything but I went through all 4 quartermasters (2 for alliance, 2 for horde) and changed what they sell based on patch. Let me know if I missed something or did something wrong.

**Comment by kamila231993** [comment]
[<a href="https://freeimage.host/i/JWGT8Zu"><img src="https://iili.io/JWGT8Zu.md.png" alt="JWGT8Zu.md.png" border="0"></a>](url)

are these items ok at the mammoth vendor?

**Comment by noisiver** [comment]
Closing this for the time being. New issues can be created for issues that might pop up related to this but as it is it's best to mark it as complete.
